toonz/sources/toonz/startuppopup.cpp

```diff
@@ -409,7 +410,13 @@ void StartupPopup::refreshRecentScenes() {
     int i = 0;
     for (QString name : m_sceneNames) {
       if (i > 9) break;  // box can hold 10 scenes
-      QString justName = QString::fromStdString(TFilePath(name).getName());
+      QString justName    = QString::fromStdString(TFilePath(name).getName());
+      QString projectName = QString::fromStdString(
+          TProjectManager::instance()
+              ->getProjectNameByScenePath(TFilePath(name))
+              .getName());
```

Comment:
I'm bit concern about that searching of project names for all 10 recent scene files may be time-consuming, especially if the files are stored in remote file server.
Instead, how about saving the project names along with the scene paths into the `RecentFiles.ini` for this purpose?